### ISSUE TITLE
Additional Interrupts

### ISSUE THREAD

**Issue by Cager1974** [Hekili/hekili#429]
I notice some "other" rotation helpers suggest additional interrupts.  I mostly notice this on my rogue.  Hekili only ever seems to suggest kick, but Kidney Shot is a viable interrupt too, and actually works many times when kick won't.  I suspect the same may be true for the Tauren War Stomp and other stun spells that can effectively interrupt spell casting.  Have only tested this for sure on the rogue, though.  I did try adding Kidney Shot as an interrupt in the interface, to no avail.  Might have done it wrong, I suppose.

All this being said, I love how much more lightweight Hekili is, and personally would not want to overtax the add-on.  I felt like the other rotation helper I was using was too bloated.

**Comment by Hekili** [comment]
Hi; there's no real reliable way to tell when a stun is going to work (or not), and if it will successfully interrupt vs. many creatures.  As such, it's potentially a DPS loss to tell people to spend time/resources on a stun that may not work.

**Comment by Cager1974** [comment]
While I understand the concern, I have seen it work.  Reliably even.  As
I'm not an addon author, I don't know for sure what the process is.  I
would assume that checking if a casting target is stunnable, and primary
interrupt is on CD, would return the result that a stun is suggested.  It's
certainly less of a DPS loss than succumbing to a Fear spell, for example.
The beauty of your addon is that the interrupt can be suggested
independently of the DPS rotation.  It can show as an option alongside the
suggested damage spell.

Again, I appreciate that Hekili is much more lightweight than Ovale and has
a much better look and interface.  Those are the main reasons I converted
in the first place, and would understand if the stun/interrupt would be too
taxing.

Thanks for getting back to me.

**Comment by Hekili** [comment]
There's no API available that tells if a target is stunnable.  If I had to guess, other addons just recommend stuns as a fallback option in general.  That's perfectly fine and achievable through a custom priority in the addon.

I may develop a feature to help make it easier to add/remove utility stuff (like interrupts, dispels) to the priority without getting into the challenging part of developing a priority.

**Comment by Cager1974** [comment]
It may indeed be a fallback suggestion when the primary interrupt is on CD. I’m surprised there’s not an API for stuns. Seems like I used have a line in the tooltip if I could Sap a target. 

I did try to add Kidney Shot into the interrupt priority, but obviously did it wrong.   It would be nice to have that extra bit of customization one day. 

Thanks for the added info and response. Much appreciated.
